include/VerificationValidation.h

```diff
@@ -9,11 +9,38 @@
 #include <QString>
 
 namespace VerificationValidation {
+    class Arg {
+    public:
+        QString argument;
+        bool isVariable;
+        QString defaultValue;
+
+        Arg(QString argument, bool isVariable, QString defaultValue){
+            this->argument = argument;
+            this->isVariable = isVariable;
+            this->defaultValue = defaultValue;
+        }
+    };
+
     class Test {
     public:
         QString testName;
         QString testCommand;
         QString suiteName;
+        QString category;
+        bool hasVariable;
+        std::vector<Arg> ArgList;
+
+        QString getCmdWithArgs() const {
+            QString cmd = testCommand;
+            for(int i = 0; i < ArgList.size(); i++){
+                cmd = cmd + " " + ArgList[i].argument;
```

Comment:
yesterday, I think Jubey mentioned something about having the user be the one to add the space?